File: packages/sdkwork-utils-python/sdkwork_utils/path.py
def join_path(*segments: str) -> str:
    cleaned = [segment.strip("/") for segment in segments if segment.strip("/")]
    return "/".join(cleaned)


def normalize_path(value: str) -> str:
    parts = [part for part in value.split("/") if part]
    joined = "/".join(parts)
    return f"/{joined}" if value.startswith("/") else joined

File: packages/sdkwork-utils-typescript/src/path.ts
export function joinPath(...segments: string[]): string {
  return segments
    .map((segment) => segment.replace(/^\/+|\/+$/g, ""))
    .filter(Boolean)
    .join("/");
}

export function normalizePath(value: string): string {
  const parts = value.split("/").filter(Boolean);
  const joined = parts.join("/");
  return value.startsWith("/") ? `/${joined}` : joined;
}
